S3/S3.py

```diff
@@ -1064,6 +1065,27 @@ def set_acl(self, uri, acl):
         response = self.send_request(request)
         return response
 
+    def set_versioning(self, uri, status):
+        headers = SortedDict(ignore_case = True)
+        headers['content-type'] = 'application/xml'
+        body = '<VersioningConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">\n'
+        body += '<Status>%s</Status>\n' % status
```

Comment:
No need to add the line breaks (\n) in the xml. This is a message for the server, not for a human.